src/dialog/actorWizardApplication.hbs

```diff
@@ -0,0 +1,76 @@
+{{log "*** actorWizard" this}}
+<form class="{{cssClass}}" autocomplete="off">
+  <div class="mainbackground"></div>
+
+  <header class="sheet-header">
+    <div class="topheaderline">
+      <h1>{{localize "RQG.ActorCreation.AdventurerCreationWizardTitle"}}</h1>
+      <h1 class="charname"><input name="name" type="text" value="{{actor.name}}" /></h1>
+    </div>
+  </header>
+
+  <div class="navbackground"></div>
+  <nav class="creation-sheet-tabs tabs" data-group="primary">
+    <a class="item norse" data-group="primary" data-tab="0-species">
+      {{{colonBreak (localize "RQG.ActorCreation.Nav.0-Species")}}}</a>
```

Comment:
I think a better way to handle the newlines is to add a css class with `white-space: pre-line;` along with adding `\n` in the translation. That way it's more flexible where you want to put the newline. There is already a class `.nowrap` so maybe a `.preservewrap` could be added. This way the extra handlebar helper isn't needed.

Reply:
Done.  This does require that everything inside the <a> tag be on the same line, and I left a comment to that effect.  I'm concerned Prettier could break this on us.